google/api/control/report_request.py

```diff
@@ -237,8 +238,8 @@ def __new__(cls,
         _validate_timedelta_arg('backend_time', backend_time)
         _validate_timedelta_arg('overhead_time', overhead_time)
         _validate_timedelta_arg('request_time', request_time)
-        _validate_int_arg('request_size', request_size)
-        _validate_int_arg('response_size', response_size)
+        _validate_int_arg('request_size', request_size, allow_zero=False)
```

Comment:
Do request and response size only refer to bodies? If so, isn't an empty request legitimate?